packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/java/summaryofsteps.md

```diff
@@ -1,7 +1,7 @@
-The following graphic shows the steps for the sign in with password and email flow:
+The following summarizes the steps involved in a sign-in flow with a password and email:
```

Comment:
Java diagram comments:

1. Missing begin() method call
2. Return step missing mention that authenticator list is also returned

Reply:
1. fixed
2. not fixing. see earlier.